Add TR-Rando information

lahm86 opened the pull request:
Added information for the randomizer project, including features and showcases.

lahm86 committed:
Add TR-Rando information

Added information for the randomizer project, including features and
showcases.

rr- reviewed (approved):
Is it OK for the project to be named "Randomizer" in the logotype and links?

rr- commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -8,14 +8,127 @@
 
+  <p>TR Rando is a randomizer tool for the classic Tomb Raider series. It was originally created for TR2, with TR1 and TR3 support later being added; in the future, TR4, TR5 and each of the expansion packs will also be supported.</p>
+
   <p>Useful links:</p>
```
Let's change this to `<h2>Useful links</h2>`.

rr- committed:
improve h2 margins

rr- committed:
improve rando header appearance

lahm86 commented:
> Is it OK for the project to be named "Randomizer" in the logotype and links?

I think this is OK. Randomizer makes it quite clear what it is and does; "TR-Rando", "the rando" etc are all aliases really. Maybe though we should keep the project name now that I think about it - so TR-Rando throughout?

rr- commented:
>Maybe though we should keep the project name now that I think about it - so TR-Rando throughout?

Yeah I'd suggest to stick to a single name.

lahm86 committed:
standardize rando naming

Use TR-Rando throughout.
Git ignore file updated to ignore local testing.

lahm86 commented:
Thanks. Pushed that now, also updated .gitignore to ignore local testing.

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+  <ProjectLogo title="TR-Rando" image="static/rando/logo.png" show_title={ true }/>
 
-  <p>Useful links:</p>
+  <p>TR-Rando is a randomizer tool for the classic Tomb Raider series. It was originally created for TR2, with TR1 and TR3 support later being added; in the future, TR4, TR5 and each of the expansion packs will also be supported.</p>
```
This sentence seems long / oddly worded. I suggest:
`TR-Rando is a randomizer tool for the classic Tomb Raider series. The tool supports TR1, TR2, and TR3. In the future, support is planned for TR4, TR5 and the expansion packs.`

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+
+  <h2>Features</h2>
+
+  <p>The randomizer can mix-up many aspects of the original levels, creating a unique experience each time you play, and at times some very interesting scenarios. Everything is optional so you can tune the outcome to your preference each time.</p>
```
`The randomizer can mix-up many aspects of the original levels, creating a unique experience each playthrough. All settings are optional, so the outcome can be tailored to player preference.`

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+  <ul>
+    <li>Change the location and type of each pickup item</li>
+    <li>Key items can also be included for an additional challenge</li>
+    <li>Create a one-item mode challenge</li>
```
Idk what this means. Maybe elaborate a little?

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+  <h4>Items</h4>
+  <ul>
+    <li>Change the location and type of each pickup item</li>
+    <li>Key items can also be included for an additional challenge</li>
```
`Key item location can also be randomized for an additional challenge` maybe?

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+
+  <h4>Secrets</h4>
+  <ul>
+    <li>Change the location of each secret (TR1 and TR3 use artefact pickups, similar to TR2)</li>
```
Does this mean like the TR2 dragons?

lahm86 replied:
Yes, so rather than TR1/3 secrets where you just get regular items, you will search for artefacts (like the gold idol could be in Caves). If you collect them all, a reward room is unlocked at the end of the level (TR2 scripts the rewards so they get added automatically).

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+    <li>Change the location of each secret (TR1 and TR3 use artefact pickups, similar to TR2)</li>
+    <li>Choose between easy and hard mode and optionally include locations that require glitch knowledge</li>
+    <li>Randomize the rewards you receive for collecting all secrets</li>
+    <li>Change the appearance of pickup sprites</li>
```
This was mentioned above in `Items`.

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+    <li>Change triggered music tracks</li>
+    <li>Change ambient music</li>
+    <li>Change the majority of sound effects</li>
+    <li>Use uncontrolled SFX mode for a truly random experience</li>
```
Maybe personal preference, but I think `uncontrolled` sounds odd. I'd just say `Randomize sound effects...`?

lahm86 replied:
There are two types: regular SFX rando, which categorises all the sounds, so you might get the magnum sound for the pistols, and apes sounding like centaurs; but in uncontrolled mode, there is no categorisation, so you might get Lara saying "nice" every time she fires or apes sounding like exploding eggs. Maybe it's just bloat here and a single "randomize sounds effects" line is fine?

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -3,19 +3,132 @@
+  <h4>Environment</h4>
+  <ul>
+    <li>Change water levels</li>
+    <li>Move keyholes, levers and switches</li>
```
Add Oxford comma? xd

walkawayy reviewed (approved):
Made some language suggestions (sorry) but it looks good.

lahm86 committed:
update rando wording

lahm86 commented:
> Made some language suggestions (sorry) but it looks good.

Thanks, appreciate the feedback. I've made those changes. For the SFX entries, I've simplified it and added a different feature to avoid confusion.

walkawayy commented on lost_artefacts/components/PageRando.jinja:
```diff
@@ -8,7 +8,7 @@
   <ProjectLogo title="TR-Rando" image="static/rando/logo.png" show_title={ true }/>
 
-  <p>TR-Rando is a randomizer tool for the classic Tomb Raider series. It was originally created for TR2, with TR1 and TR3 support later being added; in the future, TR4, TR5 and each of the expansion packs will also be supported.</p>
+  <p>TR-Rando is a randomizer tool for the classic Tomb Raider series. The tool supports TR1, TR2, and TR3. In the future, support is planned for TR4, TR5 and the expansion packs.</p>
```
Oof sorry missed an Oxford comma suggestion. Just want to make sure people don't think only TR5 had expansions. When it's actually the only one without one lol.

lahm86 committed:
update rando grammar
